index.d.ts

```diff
@@ -672,7 +673,7 @@ declare namespace Moleculer {
 		version?: string | number;
 		settings?: S;
 		dependencies?: string | ServiceDependency | Array<string | ServiceDependency>;
-		metadata?: GenericObject;
+		metadata?: GenericObject | any;
```

Comment:
It seems that metadata _can_ be `any` type (per the docs) and doesn't need to be an object.  With that said, a type union with `any` evaluates to `any`, so the `GenericObject` can be eliminated from this union.